exporter/datadogexporter/examples/k8s-chart/configmap.yaml

```diff
@@ -11,7 +11,50 @@ data:
       otlp:
         protocols:
           grpc:
-          http:
+      # The hostmetrics receiver is required to get correct infrastructure metrics in Datadog.
+      hostmetrics:
+        collection_interval: 10s
+        scrapers:
+          paging:
+            metrics:
+              system.paging.utilization:
+                enabled: true
+              system.paging.usage:
+                enabled: true
+          cpu:
+            metrics:
+              system.cpu.utilization:
+                enabled: true
+          disk:
+          filesystem:
+            metrics:
+              system.filesystem.utilization:
+                enabled: true
+          load:
+          memory:
+            metrics:
+              system.memory.utilization:
+                enabled: true
+          network:
+            metrics:
+              system.network.io.receive:
+                enabled: true
+              system.network.io.transmit:
+                enabled: true
+          processes:       http:
```

Comment:
This line looks off, is there a newline missing maybe?

Reply:
Damn. Fixed.